### PR TITLE
PDY-1121 Java Sample Accelerator should include files for creating a fat jar

### PR DESCRIPTION
**Pull request by joeeltgroth**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,4 +12,51 @@
 		<maven.compiler.source>11</maven.compiler.source>
 		<maven.compiler.target>11</maven.compiler.target>
 	</properties>
+    <dependencies>
+		<dependency>
```
Comment (dolfolife) [inline pom.xml]: Do we need this dependency for building the fat jar?

File: build.gradle [hunk 2]
```diff
@@ -1,15 +1,23 @@
 plugins {
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
 	id 'java'
 }
 
-group = 'functions'
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.vmware.functions'
```
Comment (dolfolife) [inline build.gradle]: Is not the package where the function is just `functions`?

**Comment by dolfolife** [comment]
Can we add in the https://github.com/sample-accelerators/java-functions-accelerator/blob/main/DEPLOYING.md file how we can build fat jars?

**Review by dolfolife — APPROVED** [review]

**Comment by bbtong** [comment]
@joeeltgroth @dolfolife looks like this went stale, can we get it tested and merged?

**Pull request merged**